src/dap/connection.ts

```diff
@@ -220,13 +220,16 @@ export default class Connection {
         this._telemetryReporter?.reportSuccess(msg.command, receivedTime);
       } catch (e) {
         console.error(e);
+        const format = isExternalError(e)
+          ? e.message
+          : `Error processing ${msg.command}: ${e.stack || e.message}`;
```

Comment:
Won't this end up showing the stacktrace of the ExternalError  error instance within `js-debug`, rather than the stacktrace of the originating code?

Reply:
I think you are looking at it backwards

Reply:
Right now we don't get the stack at all -

![image](https://user-images.githubusercontent.com/323878/71219623-9dfa4300-227a-11ea-80f5-e896730b9950.png)

which is suboptimal but I figured I'd wait for microsoft/vscode#87443 to do more, I think most likely we'd end up getting rid of the ExternalError code.